### PR TITLE
New suites

### PR DESCRIPTION
**Pull request by matanper**

### CODE REVIEW

File: tests/suites/test_suites.py [hunk 1]
```diff
@@ -48,116 +48,59 @@ def iris(iris_clean) -> t.Tuple[Dataset, Dataset, AdaBoostClassifier]:
...
-    arguments = (
-        dict(train_dataset=train, test_dataset=test, model=diabetes_model),
-        dict(train_dataset=train, model=diabetes_model),
-        dict(test_dataset=test, model=diabetes_model),
-    )
-
-    for args in arguments:
-        result = suite.run(**args)
-        validate_suite_result(result, expected_results='mixed')
+# @pytest.fixture()
```
Comment (matanper) [inline tests/suites/test_suites.py:51]: @yromanyshyn hey, I wasn't sure why you tested the performance suites separately, and the purpose of this fixture. does it is still needed?

File: deepchecks/suites/default_suites.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    The suite runs a set of checks that are meant to validate correctness of train-test split, including
+    integrity, drift and leakage.
+    """
+    return Suite(
+        'Train Test Validation Suite',
+        DominantFrequencyChange().add_condition_ratio_of_change_not_more_than(),
+        CategoryMismatchTrainTest().add_condition_new_category_ratio_not_greater_than(),
+        NewLabelTrainTest().add_condition_new_label_ratio_not_greater_than(),
+        StringMismatchComparison().add_condition_no_new_variants(),
+        TrainTestFeatureDrift().add_condition_drift_score_not_greater_than(),
```
Comment (noamzbr) [inline deepchecks/suites/default_suites.py:75]: put the drift first, they are more interesting

File: deepchecks/suites/default_suites.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+
+def model_evaluation() -> Suite:
+    """Create 'Model Evaluation Suite'.
+
+    The suite runs a set of checks that are meant to test model performance and overfit.
+    """
+    return Suite(
+        'Model Evaluation Suite',
+        ConfusionMatrixReport(),
```
Comment (noamzbr) [inline deepchecks/suites/default_suites.py:89]: start with the general reports (PeformanceReport, TrainTestDifference, Class performance and simple model)

File: deepchecks/suites/default_suites.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+def full_suite() -> Suite:
+    """Create 'Full Suite'.
+
+    The suite runs all deepchecks' checks.
+    """
+    return Suite(
+        'Full Suite',
+        ModelInfo(),
+        ColumnsInfo(),
+        single_dataset_integrity(),
```
Comment (noamzbr) [inline deepchecks/suites/default_suites.py:115]: reverse the order, and remove leakage as it's already in validation

**Review by noamzbr — CHANGES_REQUESTED** [review]

**Review by noamzbr — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -158,8 +158,8 @@ df_train, df_test = train_test_split(iris_df, stratify=iris_df[label_col], rando
 To run an existing suite all you need to do is import the suite and run it -
 
 ```python
-from deepchecks.suites import integrity_suite
-suite = integrity_check_suite()
+from deepchecks.suites import train_test_validation
+suite = train_test_validation()
 suite.run(train_dataset=df_train, test_dataset=df_test, check_datasets_policy='both')
```
Comment (shir22) [inline README.md:163]: fix here also the check_datasets_policy?

File: deepchecks/suites/README.md [hunk 6]
```diff
@@ -14,38 +14,14 @@
...
-  - date_leakage_suite - for datasets with a date column
-  - data_leakage_suite  - for all datasets
-  - leakage_suite - containing all three suites above
-  - overfit_suite - run all overfit checks
-  - methodological_flaws_suite - checks for all methodological flaws, including unused features
+  - single_dataset_integrity - Runs a set of checks that are meant to detect integrity issues within a single dataset.
+  - train_test_leakage - Runs a set of checks that are meant to detect data leakage from the training dataset to the test dataset.
+  - train_test_validation - Runs a set of checks that are meant to validate correctness of train-test split, including integrity, drift and leakage.
+  - model_evaluation - Runs a set of checks that are meant to test model performance and overfit.
+  - full_suite - Runs all deepchecks' checks.
```
Comment (shir22) [inline deepchecks/suites/README.md:23]: Not accurate - it runs many, but not necessarily all
(It's the "all you can get" scenario, but some checks may be irrelevant for this. e.g. when we'll have explainability checks, and various checks that requires more configurations like "segment performance")

**Review by shir22 — CHANGES_REQUESTED** [review]

File: notebooks/examples/configure_check_conditions.ipynb [hunk 7]
```diff
@@ -175,13 +175,49 @@
     {
```
Comment (shir22) [inline notebooks/examples/configure_check_conditions.ipynb:175]: <p>This is quite a long suite for the purpose of this demonstration, maybe there is a simpler/shorter one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Pull request merged**